test/e2e/json-rpc/switchEthereumChain.spec.js

```diff
@@ -53,13 +53,8 @@ describe('Switch Ethereum Chain for two dapps', function () {
         await driver.clickElement('.request-queue-toggle');
 
```

Comment:
This file assumed that window handles at certain indices would always map to the dapps that are opened. The problem is that the offscreen document can actually be one of the window handles at those indices, The solution is to return the new handle with the `openDapp` call, and then use that handle to select the right windows for the opened dapps